<a href="https://colab.research.google.com/github/mukeshy97/Business-Strategy-and-Analytics-Case-Study-/blob/main/Second_Case_Study.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **Mukesh Yadav**
Email: mukeshy3010@gmail.com  Phone no.- +917506633840

<b><h1>Second Case Study:<b><h1> 

Answers: 
First of all let's understand the above facts related to any particular organization.
1.	For membership in that organization, the fee is 20 rupees.
2.	10 rupees for extra features in the card.   
3.	With a loss rate of 3% per card. 
4.	Interest rate for each card is 15%. 
5.	Average card balance is 1000 rupees.
6.	The particular Organization should pay 6.5% of funds every month to the bank where it gets all the money. (amount of fund can change every month)
7.	A statement is issued on the 1st of every month.
8.	Credit-free period = 15 days from statement issue date ( there is no interest charge during these 15 days)

<h4>For further study we have to divide the customers into four groups.<h4> 

* **Group 1:** who pays the bill within Credit-free periods.(there is
 no interest chargeduring these 15 days)

* **Group 2:** who pays the bill in 30days after the last due date.
* **Group 3:** who pays the bill in 45days after the last due date.
* **Group 4:** who pays the bill in 60days after the last due date.

There are 1000 customers in our case study.


<h4>Q1: How much profit % is generated in 1 year?<h4>

**Scenario 1 :**

In [ ]:
intrest_rate = ((15-15)*1000*12*0.15*1000*(1/365))
print(intrest_rate)

0.0


In [ ]:
intrest_rate1 = ((30-15)*1000*12*0.15*1000*(1/365))
print(intrest_rate1) 

73972.60273972603


In [ ]:
intrest_rate2 = ((45-15)*1000*12*0.15*1000*(1/365))
print(intrest_rate2)

147945.20547945207


In [ ]:
intrest_rate3 = ((60-15)*1000*12*0.15*1000*(1/365))
print(intrest_rate3)

221917.80821917808


In [ ]:
Total_Profit = intrest_rate + intrest_rate1 + intrest_rate2 + intrest_rate3
print(Total_Profit)

443835.6164383562


<h4>So if all the member does full bill payment with no defaulter then the total Profit will be 443835.6164<h4>




**Scenario 2:**

<h4>In this scenario the interest rate is compounded quarterly so interest rate will increase only after 3 months and it will be calculated on a monthly basis. And the membership grows 5% every month and the card balance of each member also increases by 5%.<h4>

Month 1 has 1000 customers,

Month 2 has 1050 customers. i.e. 0.05*1000+1000= 1050

Month 3 has 1102 customers. i.e. 0.05*1050+1000= 1102

The balance for the first month will be 1000 Rs

2nd month - it will be an increase in 5 % every month
0.05 * 1000 + 1000 = 1050 Rs

3rd month 0.05 * 1050 +1050 = 1102.5 Rs


In [ ]:
month1 = 0
month2 = (0.05*1000+1000)
month3 = (0.05*1050+1050)
print(month1)
print(month2)
print(month3)

0
1050.0
1102.5


In [ ]:
#Group 1
interest1 = 0
print(interest1)

0


In [ ]:
#Group 2
#month 1
interest2 = ((30-15)* 1000 * 0.15) /31
print(interest2)
Total_interest1 = 1000 * interest2
print(Total_interest1)

#month 2
interest3 = ((30-15)* 1000 * 0.15) /28
print(interest3)
Total_interest2 = 50 * interest3  #50 new customers added at 5% on total 250 customers.
print(Total_interest2)

#month 3
interest4 = ((30-15)* 1000 * 0.15) /31
print(interest4)
Total_interest3 = 102 * interest4    #102 new customers added at 5% on total 250 customers.
print(Total_interest3)

72.58064516129032
72580.64516129032
80.35714285714286
4017.857142857143
72.58064516129032
7403.225806451613


In [ ]:
#Group 3
#month 1
interest5 = ((45-15)* 1000 * 0.15) /31
print(interest5)
Total_interest4 = 1000 * interest5
print(Total_interest4)

#month 2
interest6 = ((45-15)* 1000 * 0.15) /28
print(interest6)
Total_interest5 = 50 * interest6 #50 new customers added 
print(Total_interest5)

#month 3
interest7 = ((45-15)* 1000 * 0.15) /31
print(interest7)
Total_interest6 = 102 * interest7    #102 new customers added
print(Total_interest6)

145.16129032258064
145161.29032258064
160.71428571428572
8035.714285714286
145.16129032258064
14806.451612903225


In [ ]:
#Group 4
#month 1
interest8 = ((60-15)* 1000 * 0.15) /31
print(interest8)
Total_interest7 = 1000 * interest8
print(Total_interest7)

#month 2
interest9 = ((60-15)* 1000 * 0.15) /28
print(interest9)
Total_interest8 = 50 * interest9  #50 new customers added 
print(Total_interest8)

#month 3
interest10 = ((60-15)* 1000 * 0.15) /31
print(interest10)
Total_interest9 = 102 * interest10    #102 new customers added
print(Total_interest9)

217.74193548387098
217741.93548387097
241.07142857142858
12053.57142857143
217.74193548387098
22209.67741935484


In [ ]:
Total_interest_diffrent_group = Total_interest1+Total_interest2+Total_interest3+Total_interest4+Total_interest5+Total_interest6+Total_interest7+Total_interest8+Total_interest9
print(Total_interest_diffrent_group) 
Total_fund = 3317455
percentage_total_fund = 0.065*Total_fund
print(percentage_total_fund)
Total_profit = Total_interest_diffrent_group - percentage_total_fund
print(Total_profit)

504010.3686635945
215634.575
288375.79366359446


**Conclusion:** The total profit for the starting three month will be 288375.79 approx.

<h>Q2: The 3% loss rate can either mean either 3% of the outstanding balance defaults or 3% of the borrowers default on their entire balance. If the average 
balance is increased to $2000, what happens to the loss?<h>

**Answers :** In the above question we consider group 4 as borrowers default.


In [ ]:
Total_borrowers_default_1 = 1000*1000+intrest_rate3    #1000 for card balance & 1000 for customers choice
print(Total_borrowers_default_1)
Loss_1 = 1221917.80*(3/100)
print(Loss_1)
Total_borrowers_default_2 = 2000*1000+(intrest_rate3*2)    #2000 for card balance & 1000 for customers choice
print(Total_borrowers_default_2)
Loss_2 = 2443835.61*(3/100)
print(Loss_2)
Loss_2/Loss_1

1221917.808219178
36657.534
2443835.616438356
73315.0683


2.0000000081838567

<h>If the average balance is increased to $2000, then the loss will increase to 2<h>

<h>Q3: Is a borrower with a low balance more or less likely to default then a borrower with a high balance? Why ? explain clearly<h>

**Answer:** Borrowers with low balance are less likely to default compared
to borrowers with high balance because borrowers with high
balance must have high interest amount even though interest rate
is same. Interest has direct co-relation with balance and
default chances.


<h>Q4: The group company has now become unhappy with the credit card provider. it wants to start its own credit card. But for that, it will need to purchase the customers from the current credit card company. If the affilated group offered Rs 20 per account to purchase all the customers from the credit card company, what would there profit be? (assume that we are only talking about 1 year)<h> 

**Answer:** For credit card company,There are 10000 members in the beginning but because of borrower default group 4 has been eliminated so remaining members are 750.



In [ ]:
Operation_cost = 750*25
Money_paid = 750*20
Total_Profit = Operation_cost - Money_paid
print(Total_Profit)

3750


<h>So the operation cost would be =  **750*25=Rs 18750** (this amount
will be used to buy credit card customer).

Money paid to credit card company=  **750*20=Rs 15000**

**Total profit= received amount – given amount Profit = 18750-15000= Rs 3750**

This profit would be only for starting organization’s own credit
card<h>

<h>Q5: What numbers can the financial institution change to convince the affiliated group not to want to purchase, while still not going into a loss? Give a range of what the new numbers are (min and max)<h>

**Answer:** There is 6.5% fund goes to bank then the total fund as follows.


In [ ]:
Total_fund = 1000*1000    #for 1000 customer 
Fees_to_bank = Total_fund*6.5/100
print(Fees_to_bank)


65000.0


<h>So for profit of credit card company we can say that **Total intrest > fees to bank**<h>
Total intrest =  **idcP**

P= principal amount ,d= no. of days , c=no. of customer
We can’t control d,c & P but we can control i
So,  **i>65000/(1000000*d)** i.e. **i>(0.065/d)**

so for **d=30** i need to be minimum **(0.065/30)**
for i to be maxm , d need to be minimum and vice versa

<h>Q6: What is the extra cost(if any) company will have to incur if the customer delays the payment of the credit card bill after the due date?<h>

<h>**Answer:** Compnay will have profit if customer delays the
payment after the due date because of the intrest.<h>

<h>Q7: Is it beneficial for the company if the customer pays the credit card bill after the due date? (express your views as a business strategy analyst with a short example)<h>

<h>**Answer:** Of course It is beneficial for the company if customer
pays the bill after the due date but the rate of profit can
differ. For example let a customer pays the bill 5 days after
the due date then the company won’t have any profit but if the
customer pays after 15 days (credit free period) then every
extra date has interest on principle amount. Payment can also be
done in partial so we can see more variation in profit.<h>